pr-data.csv

```diff
@@ -1396,125 +1396,125 @@ https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modu
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.abortWriteRemovesUncommittedVersion,ID,Deleted,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd,
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.abortWriteReturnsTheRemovedVersion,ID,Deleted,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd,
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.addWriteReturnsNullIfNoUncommittedVersionExists,ID,Deleted,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd,
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.addWriteReturnsUncommittedVersionIfItExists,ID,Deleted,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd,
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.addWriteWithDifferentTxIdThrows,ID,Deleted,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd,
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.afterRemovalPreviousVersionRemainsAccessibleByTimestamp,ID,Deleted,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd,
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.afterRemovalReadByLatestTimestampFindsNothing,ID,Deleted,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd,
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.afterRemovalReadWithTxIdFindsNothing,ID,Deleted,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd,
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.committedMultiPageValuesAreReadSuccessfully,ID,MovedOrRenamed,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd,
```

Comment:
As per the instruction, `To use this status, please also provide some [Notes](https://github.com/TestingResearchIllinois/idoft#adding-notes) on what the test is renamed to`.
I think you also need to add the moved location in the note, but maybe you can also add a new entry with the new location.
I'm not sure, @darko-marinov, could you comment on what's the best practice here?

Reply:
Check if the method body is the same or different, as per https://github.com/TestingResearchIllinois/idoft#how-to-indicate-that-a-test-is-renamed-or-moved-to-a-different-location, to decide whether to put `Deleted` or `MovedOrRenamed`. Also add a new row as per those instructions.